domains/cyber.localplayer.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+    "domain": "localplayer.dev",
+    "subdomain": "cyber",
+
+    "owner": {
+        "email": "michaelinso12@gmail.com"
+    },
+
+    "record": {
+        "A": ["213.188.192.2"],
+        "AAAA": ["2a09:8280:1:9032:60eb:1797:5542:d0db"],
+        "CNAME": ["cyber.localplayer.dev"]
```

Comment:
Oops, you can't have a CNAME and other records present. The way CNAME works it only allows CNAME alone or other records without CNAME.